### PR TITLE
microscopy image cursor hover is pointer

### LINKED ISSUE #225: NSF Exercises 1 & 3 - the hand cursor should appear when hovering over the microscopy image
The cursor should change to the hand cursor when the user is hovering over the microscopy image in the microscope's viewfinder. This will indicate to the user that s/he can drag the image in the viewfinder to get a better look at it.

### PR DESCRIPTION
**Pull request by annagav**
Fixes #225

### CODE REVIEW

File: html_app/ui/microscopy.gss [hunk 1]
```diff
@@ -1368,6 +1368,7 @@ top: 49px;
 	bottom: -17px;
 	overflow:hidden;
 	background:black;
+	cursor: pointer;
```
Comment (pdpinch) [inline html_app/ui/microscopy.gss:1371]: I think `cursor: grab;` is more appropriate

Comment (annagav) [inline html_app/ui/microscopy.gss:1371]: I think grab is used for text selection.

Comment (pdpinch) [inline html_app/ui/microscopy.gss:1371]: That seems odd to me. Can you give me an example?

http://www.w3.org/TR/css3-ui/#cursor says `grab`:

> Indicates that something can be grabbed (dragged to be moved). Often rendered as the backside of an open hand.

If we're already using `grab` elsewhere, how about `move` ?

Comment (annagav) [inline html_app/ui/microscopy.gss:1371]: Here you can see what it looks like: http://www.w3schools.com/cssref/tryit.asp?filename=trycss_cursor

**Comment by pdpinch** [comment]
Ok. It doesn't work as advertised. How about 

`cursor: move` ?

**Comment by annagav** [comment]
Alison confirmed that she wanted the white hand with pointed finger.

**Pull request merged**